addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -27,6 +27,13 @@ $-editor-messages-margin-x: 2%;
     }
 }
 
+.oe_drop_zone_shared .oe_drop_zone {
+    &:before {
+        opacity: 0.5;
+        line-height: 20px!important;
```

Comment:
`line-height: 20px !important;`

+ tocheck with @qsm-odoo  about a good class name...
I was thinking about the opposite, while it is true that most of the use case is when drop zone is shared, a more explicit class 'xxx_not_nearest' ...

Reply:
I am not sure what `oe_drop_zone_shared` means without reviewing the PR indeed. oe_structure_not_nearest, even if not pretty would be clearer indeed :wink:

Reply:
Makes sense, I'll change it to that. The reason why it's currently called oe_drop_zone_shared is because it's specific to dropzones that are used for content shared across different pages, rather than appearing on a single one.

I can see how it's confusing though :D Changes coming up in a second

Reply:
you probably miss the first line of comment about missing space between unit and !important;